lib/ansible/modules/web_infrastructure/rundeck_acl_policy.py

```diff
@@ -55,6 +55,7 @@
             - Sets the ACL policy content.
             - ACL policy content is a YAML object as described in http://rundeck.org/docs/man5/aclpolicy.html.
             - It can be a YAML string or a pure Ansible inventory YAML object.
+extends_documentation_fragment: url
```

Comment:
Adding the options from the doc fragment manually with only `version_added: '2.10'` seems to do the trick:
```suggestion
    client_cert:
        version_added: '2.10'
    client_key:
        version_added: '2.10'
    force:
        version_added: '2.10'
    force_basic_auth:
        version_added: '2.10'
    http_agent:
        version_added: '2.10'
    url_password:
        version_added: '2.10'
    url_username:
        version_added: '2.10'
    use_proxy:
        version_added: '2.10'
    validate_certs:
        version_added: '2.10'
extends_documentation_fragment: url
```
That way, everything about these options is taken from the module fragments, except that `version_added: '2.10'` is added.

Reply:
Ah, that makes sense. Thank you! Added.